src/seedu/addressbook/data/person/Person.java

```diff
@@ -8,7 +8,7 @@
  * Represents a Person in the address book.
  * Guarantees: details are present and not null, field values are validated.
  */
-public class Person implements ReadOnlyPerson {
+public class Person implements ReadOnlyPerson ,Comparable<Person> {
```

Comment:
Incorrect white space within a statement, refer to point 12: https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Java.adoc#layout